test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-using AstroBase
```

Comment:
:confused:

Reply:
Drawbacks of coding 4:30 am in morning :laughing: